scripts/gantrymonitor/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM ubuntu
+MAINTAINER Max Burnette <mburnet2@illinois.edu>
+
+RUN groupadd -g 1006 cache_access \
+    && useradd -u 1003 -g 1006 data-mover \
+    && apt-get -y update \
+    && apt-get -y install curl python python-dev python-pip \
+    && pip install flask-restful \
+    && pip install requests \
+    && pip install globusonline-transfer-api-client \
+    && pip install python-logstash \
+    && mkdir /home/gantry \
+    && chown -R data-mover /home/gantry \
+    && chown -R data-mover /var/log
+
+COPY gantry_monitor_service.py /home/gantry/gantry_monitor_service.py
+COPY config_default.json /home/gantry/config_default.json
+
+ENV MONITOR_API_PORT 5455
```

Comment:
Can you add the volume command so you make explicit what volumes can be mounted.